assistant/src/main/java/com/ibm/watson/assistant/v1/Assistant.java

```diff
@@ -99,36 +99,61 @@
  */
 public class Assistant extends BaseService {
 
-  private static final String SERVICE_NAME = "assistant";
-  private static final String SERVICE_URL = "https://gateway.watsonplatform.net/assistant/api";
+  private static final String DEFAULT_SERVICE_NAME = "conversation";
```

Comment:
is changing this from `assistant` to `conversation` breaking?

Reply:
Wow I actually overlooked that these changed...I'll have to check and make sure that it's not.